tests/stake-pools/e2e/features/stake-pools.feature

```diff
@@ -53,3 +55,5 @@ Feature: Stake Pools Loading
     And I click "continue" button
     Then I should see step 2 of 3 screen
     And I see following label on the dialog: "You have selected [ROOT] stake pool to delegate to for Test Wallet wallet."
+
+  Scenario: Stake pools user is already delegating to are correctly displayed
```

Comment:
@DeeJayElly Scenario is not defined soo please remove / uncheck TODO task because it is not done!

<img width="671" alt="Screen Shot 2020-01-21 at 17 17 53" src="https://user-images.githubusercontent.com/18653950/72822234-0a99a080-3c72-11ea-81ae-848ab4cb1dd3.png">

Steps: 
- ensure that you have a wallet
- set fak parameters to wallet with delegating pool data OR create method to delegate stake pool. At the end you should have delegated wallet to existing stake pool
- Check if stake pool is correctly displayed

Reply:
@tomothespian is this resolved?